saas/tests/test_saas.py

```diff
@@ -145,6 +145,12 @@ def test_template_operator(self):
         if DB_INSTANCE_2 in db.list_dbs():
             db.exp_drop(DB_INSTANCE_2)
 
+        url = '/saas/1/create-fast-build'
+        r = self.url_open(url)
+        self.assertEqual(r.status_code, 200, 'User must be redirected to the build')
+        self.assertIn('fast_build_001', db.list_dbs())
+        self.assert_no_error_in_db('fast_build_001')
```

Comment:
Why don't you put it in a separate test method?

Reply:
this will increase the time to pass the tests several times because we will need to separately call the  preparing_template_next method

Reply:
Check SavepointCase

https://odoo-development.readthedocs.io/en/latest/qa/python/unittest.html?highlight=httpcase

Reply:
Is it turns out we will test the prepare_template_next method in setUpClass?

Reply:
kind of